src/index.tsx

```diff
@@ -128,19 +130,40 @@ const Content: VFC<{ serverAPI: ServerAPI }> = ({}) => {
       </PanelSectionRow>
       <PanelSectionRow>
         <ToggleField
+          bottomSeparator={autoPause ? "none" : "standard"}
           checked={autoPause}
           label="Pause on focus loss"
           tooltip="Pauses apps not in focus when switching between them."
           icon={<FaStream />}
           onChange={async (state) => {
             const settings = await backend.loadSettings();
             settings.autoPause = state;
-            await backend.saveSettings(settings);
             setAutoPause(state);
+            if (!state) {
```

Comment:
By changing the logic of the `Pause on overlay` feature in the backend you wouldn't need this anymore. As a side effect it would remember the `Also on overlay` state if `Pause on focus loss` was disabled and re-enabled.